PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -134,6 +134,13 @@
 run3_common.toModify(MicroEventContent, outputCommands = MicroEventContent.outputCommands + _run3_common_extraCommands)
 # --- 
 
+_pp_on_AA_extraCommands = [
+    'keep patPackedCandidates_hiPixelTracks_*_*',
+]
```

Comment:
mind #30253 
it has a change in the same lines; since it's already signed, I expect that this PR will need to be rebased after #30253  is merged.

Reply:
Sure, I can do a rebase when #30253 is merged. Thanks!